CONTRIBUTING.md

```diff
@@ -36,61 +34,210 @@ $ sudo dnf install device-mapper-devel btrfs-progs-devel
+
 ```
 mkdir -p $GOPATH/src/github.com/openshift
 git clone https://github.com/openshift/ansible-service-broker.git $GOPATH/src/github.com/openshift/ansible-service-broker
 cd $GOPATH/src/github.com/openshift/ansible-service-broker
+# Install or update project dependencies
 make vendor
 ```
 
-## Setup
+## Deploy an OpenShift Origin Cluster with the Ansible Service Broker
+
+At some point, when contributing to the Ansible Service Broker project (or just checking it out),
+you are going to want to have a cluster to integrate with. There are two primary methods for
+starting an OpenShift Origin Cluster with the Ansible Service Broker:
+
+1. Use [fusor/catasb to gain more control over the environment](https://github.com/fusor/catasb).
+1. Use [run_latest_build.sh](../scripts/run_latest_build.sh) to create a cluster with the service
+   catalog enabled and the Ansible Service Broker running. We covered this before in
+   ["Getting Started with the Ansible Service Broker"](README.md#getting-started-with-the-ansible-service-broker).
+
+   ```
+   cd $GOPATH/src/github.com/openshift/ansible-service-broker
+   bash -x scripts/run_latest_build.sh
+   ```
+
+**Congratulations**, you should now have a running OpenShift Origin Cluster with the Service Catalog
+and Ansible Service Broker running inside. If you are having issues, have a look at
+[fusor/catasb](https://github.com/fusor/catasb) for some troubleshooting steps.
+
+### Reset OpenShift Origin Cluster
+
+**NOTE** if you ever want to reset your environment, look at [fusor/catasb](https://github.com/fusor/catasb)
+because that project makes it as easy as `./reset_environment.sh`, **or** take the cluster down and `run_latest_build.sh`
+again:
+
+```
+oc cluster down
+cd $GOPATH/src/github.com/openshift/ansible-service-broker
+./scripts/run_latest_build.sh
+```
+
+
+# Build the Ansible Service Broker
+
+Building the Ansible Service Broker is as simple as running `make build` from the root
+of the project:
+
+```
+cd $GOPATH/src/github.com/openshift/ansible-service-broker
+make build
+```
+
+Now you can [run your broker locally](#run-your-broker-locally) with `make run` or
+[package your broker using docker](#package-your-broker-using-docker) with `make build-image`.
+
+## Package Your Broker Using Docker
+
+You can also package your built Ansible Service Broker binary into a Docker image through
+`make build-image`.
+
+```
+cd $GOPATH/src/github.com/openshift/ansible-service-broker
+export ORG=${YOUR_DOCKERHUB_ID}
+export TAG=my_custom_tag
+make build-image
+```
+
+If you have a look at the [Dockerfile](build/Dockerfile-localdev) used when building a Docker
+image with the broker, there are a few things worth pointing out:
+
+- The `broker` image is copied to `/usr/bin/asbd`.
+- The home directory in the image is `/opt/ansibleservicebroker`.
+- The directory `/etc/ansible-service-broker` is where, by default, the broker's configuration
+  file is mounted when deployed in OpenShift. Looking at the
```

Comment:
CHANGE: "The directory `/etc/ansible-service-broker` is where, by default, the broker's configuration file is mounted when deployed in OpenShift."

-> 

"The directory, `/etc/ansible-service-broker`, is where the broker's configuration file is mounted, by default, when deployed in OpenShift."